modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -283,6 +283,7 @@ def veteran_phone_is_not_too_long
 
     # validation
     def informal_conference_rep_name_and_phone_number_is_not_too_long
+      return if pdf_version == 'v2'
```

Comment:
this is my thought process...not the best solution, but we need to skip some of the model validations for v2